### DISCUSSION TITLE
Vacuum Heads

### DISCUSSION THREAD

**Discussion by Irev-Dev** [zalo/CascadeStudio#46] [Show and tell]
I had a go a recreating [Zivbot's excellent OpenScad script](https://t.co/RtcU550ifU?amp=1) in CascadeStudio 

[Up here](https://t.co/6D8ZCg68eL?amp=1)

![vacuum-heads](https://user-images.githubusercontent.com/29681384/101886653-e7f8d900-3bef-11eb-9d5f-70f80d9ff245.jpg)
![carbon](https://user-images.githubusercontent.com/29681384/101886659-eaf3c980-3bef-11eb-8c90-94c6779e0488.png)
![vacuum_original](https://user-images.githubusercontent.com/29681384/101886674-ef1fe700-3bef-11eb-81dd-4ab96f2b5183.jpg)

[Tweeted it too](https://twitter.com/IrevDev/status/1337138024988041216)

**Comment by zalo** [comment]
These are quite beautiful shapes; a tour-de-force of the Lofting and Sketching features!  Thank you for posting them here!

**Reply by Irev-Dev** [reply]
Your welcome. It's good github discussions are here.
I'll keep posting new creations here. Definitely good to have a bit of a showcase.
